### DISCUSSION TITLE
Troubles setting up OT network

### DISCUSSION THREAD

**Discussion by OpalWatcher** [openthread/openthread#9660] [Q&A] [answered]
hello everyone
My goal is to set up an OpenThread network and control my Nanoleaf light using the network.

I'm in the process of creating an OpenThread network from scratch. however facing many issues, mainly due to the tutorials on openthread.io being unclear at times.  If anyone has experience with this, I'd love to hear your tips.

based on the documentation, this is the hardware Im using: 
raspberry pi 3 
2 nordic nRF52840DK
and a nRF52840 dongle

Once the OpenThread network is up and running, I want to connect my Nanoleaf light to it f. Are there specific steps or considerations when it comes to integrating Nanoleaf lights with OpenThread?

**Comment by jwhui** [comment]
The primary challenge in configuring your own Thread Border Router to work with Nanoleaf is ensuring that it is configured with the same Thread Network Credentials as what Nanoleaf expects.

For now, the only way to do this is via the [Thread Network SDK for Android](https://developers.home.google.com/thread) and [iOS Thread APIs](https://developer.apple.com/documentation/threadnetwork/). Note that this will require a mobile app to integrate with those APIs.

**Reply by OpalWatcher** [reply]
Thank you for the reply, so I should setup a OTBR without a commissioner or joiner and use the SDK to set the network up with the credentials needed for the nanoleaf? 
My apologies if my question is misguided

**Reply by jwhui** [reply, marked as answer]
Nanoleaf devices to not use Thread's Mesh Commissioning Protocol, so the Commissioner / Joiner functions are not applicable. Instead, Nanoleaf devices use the Matter / HomeKit commissioning methods.

**Reply by OpalWatcher** [reply]
Noted, thank you very much

**Reply by xjshiMax** [reply]
I have the same question. What else is there to add a light bulb to the OTBR without using nanoleaf

**Reply by jwhui** [reply]
This question is specifically about setting up Nanoleaf devices.
